src/sonic-config-engine/minigraph.py

```diff
@@ -2068,6 +2140,18 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
             'client_crt_cname': 'client.restapi.sonic'
         }
     }
+    if devices[hostname]["type"] == "BmcMgmtToRRouter":
+        dhcp_server_ipv4, dhcp_server_ipv4_port, dhcp_server_ipv4_customized_options = \
+            generate_ipv4_dhcp_server_config(devices, link_ports, port_alias_map, vlan_intfs, vlan_members,
+                                             rack_mgmt_map)
+        for vlan_name, _ in results["VLAN"].items():
+            # For BmcMgmtToRRouter, treat docker ip as new dhcp_server
+            results["VLAN"][vlan_name]["dhcp_servers"] = [DHCP_SERVER_IP]
+        if dhcp_server_ipv4 is not None:
+            results['DHCP_SERVER_IPV4'] = dhcp_server_ipv4
+            if dhcp_server_ipv4_customized_options:
+                results['DHCP_SERVER_IPV4_CUSTOMIZED_OPTIONS'] = dhcp_server_ipv4_customized_options
+            results['DHCP_SERVER_IPV4_PORT'] = dhcp_server_ipv4_port
```

Comment:
This is a new table in ConfigDB. Please onboard it with golden config generation pipeline. We are blocking new tables coming into minigraph parser.